collectors/hra/parser.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from .record import Record
+from datetime import date
+import logging
+import six
+logger = logging.getLogger(__name__)
+
+
+def parse_response(response, endpoint, from_date, to_date, errors, success, conn):
+    for application in response.json():
+        try:
+            data = {}
+            data['last_updated'] = date.today().strftime('%Y-%m-%d')
+            data['api_date_from'] = from_date.strftime('%Y-%m-%d')
+            data['api_date_to'] = to_date.strftime('%Y-%m-%d')
```

Comment:
> I'm not fully into HRA API for now but curious why we need to store request attributes as record attributes? If not it also could be removed from function args.

@roll 
The API takes `From` and `To` dates that aren't present in the data. As I understand these dates are used to request data that has been added/updated in the database in that period. I wanted to have something handy for when the collector restarts, can we log `From Date` instead, would this be useful? I will have a look at `Updated Date` and `Publication Date` and see if these are close enough to `From Data` so that when restarting the collector will start according to the `Updated Date` field (with some offset so that it won't miss data) .